interactive-bots/helloworld/sColin16/notes.md

```diff
@@ -0,0 +1,20 @@
+Problems I faced:
+=================
+
+1. **Obtaining the zuliprc configuration on the dev enviornment** - Although I was able to download the file onto my local file system, I was 
+unable to use the configuration file in the dev enviornment, because it was not there. To solve this problem, I used the scp command from my 
```

Comment:
We try to keep documentation lines <= 80-100 characters; can you shorten some of the very long lines?